pkg/api/unversioned/types.go

```diff
@@ -162,6 +162,11 @@ const (
 	// Status code 409
 	StatusReasonConflict StatusReason = "Conflict"
 
+	// StatusReasonGone means the item is no longer available at the server and no
+	// forwarding address is known.
+	// Status code 410
+	StatusReasonGone StatusReason = "Gone"
```

Comment:
Right, it's totally fine to add this status code to the api. The thing I was talking about was the range of codes we could possibly return from a given request. I don't think we consider that part of the API, or at least it's not written down places. I could be wrong?